src/planwise/component/scenarios.clj

```diff
@@ -40,15 +42,33 @@
       (update :changeset edn/read-string)))
 
 (defn create-initial-scenario
-  [store project-id]
-  ;; TODO schedule demand computation
-  (db-create-scenario! (get-db store)
-                       {:name "Initial"
-                        :project-id project-id
-                        :investment 0
-                        :demand-coverage nil
-                        :changeset "[]"
-                        :label "initial"}))
+  [store project]
+  (let [project-id  (:id project)
+        scenario-id (:id (db-create-scenario! (get-db store)
```

Comment:
db-create-scenario should set the initial state to some value for scheduled or running